setup.py

```diff
@@ -17,6 +17,7 @@
         "more_itertools",
         "nltk",
         "torch",
+        "open_clip_torch"
```

Comment:
needs a comma "," at the end of `open_clip_torch`

Reply:
done